extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -14,23 +14,93 @@
  * limitations under the License.
  */
 
+.i-amphtml-story-interactive-slider-container {
+  --percentage: 0%;
+}
+
 .i-amphtml-story-interactive-slider-input-container {
+  display: flex !important;
   background: var(--i-amphtml-interactive-option-background-color) !important;
   border-radius: 2em !important;
   box-shadow: var(--i-amphtml-interactive-component-shadow) !important;
+  align-items: center !important;
+  justify-content: center !important;
+  height: 100% !important;
+  width: 100% !important;
+  padding: 0em 1em !important;
+  box-sizing: border-box !important;
+  position: relative !important;
+}
+
+.i-amphtml-story-interactive-slider-bubble {
+  position: absolute !important;
+  transform: translate(-50%, -50%) scale(0) !important;
+  width: 2.8em !important;
+  height: 2.8em !important;
+  top: 50% !important;
+  left: var(--percentage) !important;
+  text-align: center !important;
+  line-height: 2.9em !important;
+  z-index: 2 !important;
+  font-weight: 500 !important;
+  color: white !important;
+}
+
+.i-amphtml-story-interactive-slider-bubble.show {
+  transform: translate(-50%, -155%) scale(1) !important;
+}
+
+.i-amphtml-story-interactive-slider-bubble::after {
+  position: absolute !important;
+  content: '' !important;
+  box-sizing: border-box !important;
+  height: 100% !important;
+  width: 100% !important;
+  background: var(--interactive-accent-color) !important;
+  z-index: -1 !important;
+  left: 50% !important;
+  transform: translateX(-50%) rotate(45deg) !important;
+  border-radius: 50% 50% 0% 50% !important;
+  box-shadow: 0em 0em 0.5em rgba(0, 0, 0, 0.25) !important;
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-bubble {
+  font-weight: 700 !important;
+  transform: translate(-50%, -50%) scale(0.8) !important;
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-bubble::after {
+  cursor: initial !important;
+  border: 0.125em solid white !important;
+  border-bottom-right-radius: 50% !important;
 }
 
 .i-amphtml-story-interactive-slider-input {
   -webkit-appearance: none !important;
-  width: calc(100% - 2em) !important;
+  font-size: inherit !important;
   height: 1em !important;
-  margin: 1em !important;
-  background: transparent !important;
-  position: relative !important;
+  width: calc(100% + 1.5em) !important;
+  margin: 0em -0.75em !important;
```

Comment:
Negative margins are hacky. We need to try to stay away from doing things like this if possible. I understand that it's necessary when doing this approach of compensating for the thumb size by overriding margins of a wrapper and container.

Reply:
Following up on this based on our conversation.
I see now how the negative margins are working and this seems like a totally valid approach. 
Thank you for explaining!